mappings/net/minecraft/util/NonNullList.mapping

```diff
@@ -0,0 +1,23 @@
+CLASS fj net/minecraft/util/NonNullList
+	FIELD a delegate Ljava/util/List;
+	FIELD b defaultValue Ljava/lang/Object;
```

Comment:
`fallback`. Since this is no longer `DefaultedList`, a `defaultValue` appears weird.

Reply:
Resolved by 20b995c